support/ruby/collins-notify/Gemfile

```diff
@@ -7,10 +7,10 @@ gem 'mail',           '~> 2.4.4'
 gem 'nokogiri',       '~> 1.5.2'
 
 group :development do
-  gem "rspec", "~> 2.12.0"
-  gem "yard", "~> 0.8.3"
-  gem "rdoc", "~> 3.12"
-  gem "bundler", "~> 1.2.0"
-  gem "jeweler", "~> 1.8.4"
-  gem "simplecov"
+  gem 'i18n',         '< 0.7.0'	# versions above 0.7.0 require ruby 1.9.3
```

Comment:
Do you want to require ruby < 1.9.3 too in this Gemfile?
You can also test the version of ruby and adjust the version based on that.